packages/core/package.json

```diff
@@ -44,6 +44,7 @@
     "@backstage/cli": "^0.1.1-alpha.3",
     "@backstage/test-utils": "^0.1.1-alpha.3",
     "@backstage/theme": "^0.1.1-alpha.3",
+    "@storybook/addon-storysource": "^5.3.18",
```

Comment:
I wish I didn't have to add this storybook dependency here since we separated them to `packages/storybook`. But, this particular addon would not compile for `pakcages/core/**` unless we install it here too.

cc @Rugvip 

Here is the build error log without installing it here https://pastebin.com/X7FnaJcK

Since, it is only a `devDependency`, it might be safe to add? But, I'd love feedback on this.

Reply:
Bit unfortunate if we need this here, but not the end of the world. Imo ok to merge with this and then do some further investigation. If we merge with this in, could you create an issue to investigate it?

Reply:
On it.

Reply:
Created the issue https://github.com/spotify/backstage/issues/557

Thanks for your review @Rugvip ! 🙇